docs/nx/nx_monorepo.md

```diff
@@ -106,9 +107,11 @@ Visit the [NX Example](../stackscli/about.md) for the complete instructions/exam
 
 :::
 
-### Option 2: Create stacks workspace generator
+#### Option 2: Create stacks workspace generator
 
-You can scaffold a brand new Stacks and Nx workspace through the **@ensono-stacks/create-stacks-workspace** package.
+You can scaffold a brand-new Stacks and Nx workspace using the **@ensono-stacks/create-stacks-workspace** package.
+
+This approach is ideal if you want to use Stacks to build your frontend web applications but already have an existing infrastructure set up without Stacks.
```

Comment:
Worth adding a note to say that it's still possible to create a workspace with infrastructure through this option, however you would need to pass in the CLI arguments listed on this page: https://stacks.amido.com/docs/nx/create-stacks-workspace/ensono-stacks-create-stacks-workspace#command-line-arguments

Reply:
I've updated the description for "Option 2" now. Sorry, I squashed the commits without thinking, so the update is part of the one commit.